pandas/tests/frame/test_operators.py

```diff
@@ -530,6 +530,14 @@ def test_comp(func):
         test_comp(operator.ge)
         test_comp(operator.le)
 
+    def test_strings_to_numbers_comparisons_raises(self):
+        # GH 11565
+        df = DataFrame(
+            {x: {"x": "foo", "y": "bar", "z": "baz"} for x in ["a", "b", "c"]}
+        )
+        with pytest.raises(TypeError):
+            df > 0
```

Comment:
Can you `pytest.mark.parameterize` over all the operators? I think we have a fixture.

Reply:
Thanks, @mroeschke - now using parameterize to go over `["lt", "le", "gt", "ge", "eq", "ne"]`.